### ISSUE TITLE
Chunks not loading

### ISSUE THREAD

**Issue by ajthemacboy** [softpak/HOSE#4]
I just tested the HOSE release on my private server and found that chunks don't load when I login. I've teleported myself around via console and I've relogged and restarted several times.

Here's my plugins: `AsyncWorldEditInjector, AntiWDL, WorldEdit, AsyncWorldEdit, Essentials, VoxelSniper, VoidWorld, PermissionsEx, PlugMan, LagMeter, IPLogger, Modifyworld, SpawnJoin, WorldBorder, PlotSquared, EssentialsSpawn, Multiverse-Core, Sunny, Voxelight-Place, WorldGuard`

I have Spigot's chunks per tick set to 0, and I've tweaked a couple other settings, but nothing I think would effect the server like this.

This only occurs with HOSE.

**Comment by softpak** [comment]
Is there any error in log file?

**Comment by ajthemacboy** [comment]
Nope, nothing in console or logs. The server isn't actually freezing, commands work fine, just chunks aren't loading.

**Comment by softpak** [comment]
I found the problem. Wrong sync object. You can download the latest release now.

**Comment by ajthemacboy** [comment]
I'm afraid this didn't fix my issue. Chunks still aren't loading.

Your commit says you fixed chunk generation issues, but my chunks are already generated; they just aren't loading.

**Comment by softpak** [comment]
Alright. I will try these plugins and fix it as fast as I can.

**Comment by ajthemacboy** [comment]
Alright, thanks :)  I can eliminate these plugins from your testing as they don't make any significant modifications: `Sunny, Voxelight-Place, VoidWorld, AntiWDL, IPLogger`, and probably `AsyncWorldEdit`.

**Comment by softpak** [comment]
Do you use Voxelight-Place v1.1.7? It uses a deprecated method.
Remove this plugin and try this build I attached.

**Comment by ajthemacboy** [comment]
The problem still occurs with Voxelight-Place uninstalled and the latest build.

**Comment by softpak** [comment]
May I know all these plugins` version? I downloaded them from bukkit and spigot website.
Let me list all the versions of the plugins.

**Comment by softpak** [comment]
I have a result for these plugins you listed.
Please read the previous comment.
Some plugins will jump up some errors but the server works fine.

**Comment by ajthemacboy** [comment]
I just tested without ANY plugins installed on the server and this issue still occurs. Perhaps I can send you some of the server configuration files and let you look at them?

**Comment by ajthemacboy** [comment]
Here are various config files: https://www.dropbox.com/s/veu2u9lxju37gjx/Hose.zip?dl=0

**Comment by softpak** [comment]
Will...
Your can not set both of max-tick-time to 0!!!
Like this.
`max-tick-time:`
      `tile: 0`
      `entity: 0`

These 2 numbers must be over then 0 to make tiles and entities tick.
It means when your world`s tick time over then the number you set. (in ms)
The server will stop ticking and then wait for the next tick.
It causes a loop that just do nothing.
So leave them to the default number as 50.

**Comment by ajthemacboy** [comment]
I set those to 0 because I want my server to be frozen. It's a private build server so I don't want any grass growth or entity movement or anything. I also have another server running on that machine so I try to squeeze all the performance out of it I can.

Those configuration files work fine in normal Spigot and PaperSpigot so I don't know why they would be the problem in HOSE, unless HOSE is just very different in the way it works..?

I will test the values at 50 and tell you if it works.

**Comment by ajthemacboy** [comment]
It was those values that are causing the problem. I've set them to 1 each to effectively freeze the server without disabling HOSE and it seems to work fine.

Thanks for your help, I don't see any need to fix this as it's not a real big issue.

Offtopic question though, I'd still like to be able to patch normal Spigot so I can keep the latest up-to-date builds. Can you provide any more help as to patching instructions? Possibly instructions that don't require an IDE?

Also, will you support 1.9?

**Comment by softpak** [comment]
I just have some bugs to deal with.
The noob tools is not that fast.
When I am done with 1.8.8.
I will start to patch 1.9.
So it may took a few weeks.
